v1.4.0.md

```diff
@@ -0,0 +1,285 @@
+- [GUI] [FEATURE] Host local storage support [#6057](https://github.com/harvester/harvester/issues/6057)
+- [FEATURE] Support Longhorn v2 data engine - Experimental [#5274](https://github.com/harvester/harvester/issues/5274)
+- [GUI] [FEATURE] Support Longhorn v2 data engine - Experimental [#5953](https://github.com/harvester/harvester/issues/5953)
+- [FEATURE] Support for iscsi boot disks [#5150](https://github.com/harvester/harvester/issues/5150)
+- [FEATURE] Improving Harvester Maintenance Mode [#5069](https://github.com/harvester/harvester/issues/5069)
+- [FEATURE] Add Fleet support with Harvester [#5018](https://github.com/harvester/harvester/issues/5018)
+- [FEATURE] Allow users to set count/size limit on snapshots [#4478](https://github.com/harvester/harvester/issues/4478)
+- [GUI] [FEATURE] Allow users to set count/size limit on snapshots [#6054](https://github.com/harvester/harvester/issues/6054)
+- [FEATURE] QA Cluster Improvements [#3264](https://github.com/harvester/harvester/issues/3264)
+- [FEATURE] Add volume encryption support for Harvester [#3129](https://github.com/harvester/harvester/issues/3129)
+- [GUI] [FEATURE] Add volume encryption support for Harvester [#6493](https://github.com/harvester/harvester/issues/6493)
+- [FEATURE] Optimize upgrade `image preloading` [#3059](https://github.com/harvester/harvester/issues/3059)
+- [FEATURE] bot to auto add releases when milestone is set [#2896](https://github.com/harvester/harvester/issues/2896)
+- [FEATURE] Scheduling of VM Backups [#2756](https://github.com/harvester/harvester/issues/2756)
+- [GUI] [FEATURE] Scheduling of VM Backups [#6058](https://github.com/harvester/harvester/issues/6058)
+- [FEATURE] Distribute disk volume replicas across zones [#2323](https://github.com/harvester/harvester/issues/2323)
+- [FEATURE] Support CPU Pinning [#2305](https://github.com/harvester/harvester/issues/2305)
+- [GUI] [FEATURE] Support CPU Pinning [#6056](https://github.com/harvester/harvester/issues/6056)
+- [FEATURE] Restore vm backup to other harvester cluster [#2237](https://github.com/harvester/harvester/issues/2237)
+- [GUI] [FEATURE] Restore vm backup to other harvester cluster [#6371](https://github.com/harvester/harvester/issues/6371)
+- [FEATURE] Need to support RWX of Harvester CSI Driver [#1992](https://github.com/harvester/harvester/issues/1992)
+- [FEATURE] USB passthrough support [#1710](https://github.com/harvester/harvester/issues/1710)
+- [FEATURE] Support Bundle timeout enhancement  [#1646](https://github.com/harvester/harvester/issues/1646)
+- [FEATURE] Support setting hostname via DHCPv4 client [#1444](https://github.com/harvester/harvester/issues/1444)
+- [ENHANCEMENT] [GUI] Show warning message for Longhorn V2 unsupported actions [#6989](https://github.com/harvester/harvester/issues/6989)
+- [backport v1.4] [ENHANCEMENT][GUI] Move enable CPU Pinning checkbox position in template config page [#6937](https://github.com/harvester/harvester/issues/6937)
```

Comment:
This is the first release for 1.4. Why do we have a backport issue here?  This might be related to issue handling.

Reply:
Thanks for the context. Sounds good.